frontend/components/SocialMediaShareIcon.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="cursor-pointer"
```

Comment:
This needs to be on the next line, which Prettier should do this. Do you have the [extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) installed, @GrandSuccess87?

Reply:
Yep, I do have it installed, I see now it's not used by default when I format the doc. Makes a big difference :D